### ISSUE TITLE
Panel: Narrator focus does not moves to the 'Close' button while backward navigation in scan mode by using up/down arrow keys.

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10782]
**User Experience:**
Narrator users will get confused while navigating in scan mode by using up or down arrow keys, if the focus does not lands on Close button.

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 18999.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/panel
Edge Browser Version:
Microsoft Edge: 44.18999.1.0
Microsoft EdgeHTML: 18.18999

**Usages:** 
All usages.

**Repro Steps:**
1. Open https://developer.microsoft.com/en-us/fabric#/controls/web/panel in Edge and turn on narrator.
2. Navigate to 'Panel' control by using Tab key and activate it.
3. Dialogue pane will be appeared.
4. Navigate to 'Small Panel, Anchored Left, Fixed Width' variant by using Tab key and activate it.
5. Switch narrator into scan mode.
6. Now verify the issue while backward navigation by using up arrow keys on the pane.

**Actual Result:**
Narrator focus does not lands on Close button. First It lands on the 'Small Panel, Anchored Left, Fixed Width' and again it lands on the same but narrator announces 'Close' button.

**Expected Result :**
Narrator focus should land on 'Close' button and focus should not land on 'Small Panel, Anchored Left, Fixed Width' twice.

**Note:**
Working fine with NVDA Screen reader.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7Bf98f4235-b6bd-47d3-8da4-f28b9b6fbfa8%7D&action=view&wdAccPdf=0&wdparaid=F2F4701&cid=62a0959f-b7a6-4e39-8a33-f8ef543e9cc0

**Attachment:**
[Focus does not lands on close button in scan mode while backward navigation.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3712146/Focus.does.not.lands.on.close.button.in.scan.mode.while.backward.navigation.zip)

**Comment by khmakoto** [comment]
Hi @999-Dev, this feels like a problem with either Edge or Narrator since the focus is actually going to the close button as shown by Narrator saying `button close` but the visual indicator staying on the text below. I've reached the Narrator team to ask for clarification and will update this issue accordingly.

**Comment by khmakoto** [comment]
Hi @999-Dev, the Narrator team has replied back to me and this seems to be by design on Narrator and Edge because the calculation of bounding rectangles is an approximation.

You can read the full answer from the team below:

>The issue is both on Narrator and Edge.
>
>Bounding rectangles are provided by applications that provide content. Narrator trusts received bounding rectangles to be accurate and represent bounds of content that it reads.
>
>However, in case of web browser, calculating bounding rectangles for various slices of content is often inaccurate and due to that, you can often see that Narrator's highlight does not cover what you think it should (this can even happen for the same slices of content [text content ranges] achieved through different API call steps).
>
>That said, there are situations where Narrator could do better by trying to -- out of other possible solutions -- adjust the bounding rectangle from the application based on what Narrator thinks should be read.
>
>In this specific scenario, since the Button is supposed to be read and nothing else, Narrator could safely go for its bounding rectangle. Narrator does not do it due to performance reasons (adjustments may require more cross-process communication with the application).
>
>Hopefully, those reasons will stop being a factor in a near future and thanks to that the highlight will become slightly less of an approximation.
>
>**The important note to make:** Highlight is _an approximation_ of where Narrator is. It does not reflect the exact and accurate focus.
>
>In your case, Narrator lands on the Button in both directions and allows interacting with it despite differences in the highlight.

**Comment by 999-Dev** [comment]
Hi @khmakoto,

Agreed, But this issue is repro'ing through out the controls or might be other Fabrics also. It will be good practice if narrator focus and announcement are in sync. Because here what we observe that focus is on text and narrator announces 'Close' button.

Thanks,

**Comment by khmakoto** [comment]
@999-Dev I understand your point, but there is not a whole lot we can do here since there isn't anything in how we're developing the components that is causing this, but rather the behavior of Narrator on Edge itself. Hence the reason I'm classifying this as `External`.
